libvast/vast/transform.hpp

```diff
@@ -45,13 +66,15 @@ class transform {
   /// Sequence of transformation steps
   std::vector<transform_step_ptr> steps_;
 
-  /// If all steps of this transform have specialized arrow handlers,
-  /// we can save all intermediate (de)serialization steps.
-  /// NOTE: This is ignored if VAST is built without arrow support.
-  bool arrow_fast_path_;
-
   /// Triggers for this transform
   std::vector<std::string> event_types_;
+
+  /// Transformed slices
+  batch_queue to_transform_; // FIXME: write doc
+
+  offset_range range_;
+
+  void clear_slices();
```

Comment:
Missing API docs.

Why would I need to manually clear the slices?

Reply:
One possible sequence of action: `new transform();` `add(slice1);` `add(slice2);` `finish();` `add(slice3);` `finish();` `delete transform`

The transform`add()` has to store information related to the added slices. When `finish()` is called, these resources must be released before starting the next add-finish run.

The `clear_slices()` function is not needed though, it is used only in one place, so I have removed it.